.github/workflows/codecov.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         run: make cover_profile
 
       - name: Upload coverage reports to Codecov
-        uses: codecov/codecov-action@v4.0.1 # https://github.com/codecov/codecov-action
+        uses: codecov/codecov-action@v4.5.0 # https://github.com/codecov/codecov-action
```

Comment:
This upgrade should fix the coverage upload from fork originated PRs [issue](https://github.com/zowe/zowe-client-python-sdk/issues/290) which broke coverage uploads recently